.github/workflows/release.yaml

```diff
@@ -53,6 +53,13 @@ jobs:
           CACHE_TO=type=gha,scope=eraser-node,mode=max \
           ERASER_IMG=${{ env.REGISTRY }}/${REPO}:${TAG}
 
+      - name: Build eraser for Windows
+        run: make docker-build-eraser-windows \
+          CACHE_FROM=type=gha,scope=eraser-node \
+          CACHE_TO=type=gha,scope=eraser-node,mode=max \
+          OSVERSION=1809 \
```

Comment:
I think we need to build multiple OSVERSIONs here

`ALL_OSVERSIONS.windows := 1809 1903 1909 2004 ltsc2022`

example: https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/Makefile#L154-L156

Reply:
Cool, wasn't sure since it looked like there was only a single arch build for linux. Updated!

Reply:
for Linux, we just added multi-arch https://github.com/Azure/eraser/pull/155 however, windows is a special case since it needs to match host version